src/vs/editor/contrib/codeAction/browser/codeAction.ts

```diff
@@ -319,6 +321,7 @@ export async function applyCodeAction(
 					: nls.localize('applyCodeActionFailed', "An unknown error occurred while applying the code action"));
 		}
 	}
+	accessibilitySignalService.playSignal(AccessibilitySignal.codeActionApplied);
```

Comment:
I considered adding a timeout here before this is played to ensure there's a delay but think it's ok without that based on my testing. We'll get feedback.

Reply:
actually, do think that's a good call. will add a comment though to explain it.